packages/jsx/src/index.ts

```diff
@@ -125,8 +127,15 @@ export const reatomJsx = (ctx: Ctx, DOM: DomApis = globalThis.window) => {
         key = key.slice(5)
       }
       if (val == null || val === false) element.removeAttribute(key)
-      else if (val === true) element.setAttribute(key, '')
-      else element.setAttribute(key, String(val))
+      else {
+        val = val === true ? '' : String(val)
+        /**
+         * @see https://measurethat.net/Benchmarks/Show/54
+         * @see https://measurethat.net/Benchmarks/Show/31249
+         */
+        if (key === 'class' && element instanceof HTMLElement) element.className = val
```

Comment:
Should we handle className too? Btw, don't forget to change the types
```suggestion
        if ((key === 'class' || key === 'className') && element instanceof HTMLElement) element.className = val
```